### ISSUE TITLE
The readme in AWS typescript lab 2 step assume dev stack already exists

### ISSUE THREAD

**Issue by opentrail** [joeduffy/infrastructure-as-code-workshop#39]
If you immediately follow on from lab 1 to lab 2, Provisioning EC2 Instances, the pulumi dev stack will have been deleted in lab 1 step 7 and using pulumi up in lab 2 step 1 will fail. 
The README.md file needs to include steps to re-create the dev stack.
infrastructure-as-code-workshop/labs/aws/typescript/lab-02/

**Comment by stack72** [comment]
Hi @opentrail 

Thanks for the issue here - I have tried to add these lines:

```
> This lab assumes you have a project set up and configured to use AWS. If you don't yet, please complete parts [1](../lab-01/01-creating-a-new-project.md) 
>and [2](../lab-01/02-configuring-aws.md) of lab-01.
```

If you feel this could be more clear then I am happy to make that happen

Paul

**Comment by opentrail** [comment]
Hi Paul,

Think it needs to be a bit more specific and include the folder structure for each lab. I haven't cloned the repo - I'm just following the steps in the Readme on my command line. 
Were you intending that the same folder is used? That produces an error if you re-run through lab 1 / step 1 so the operation is not idempotent.

Think I would include the steps again in a new folder.

Hope that helps.

**Comment by stack72** [comment]
Makes sense to me - thanks for the clarification @opentrail - I will get it added!
